src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/Tab.java

```diff
@@ -143,6 +148,18 @@ public void addTabClickedCallback(CallbackP<Tab> onClickCallback) {
 
 	@Override
 	public void onTabClicked() {
+		if (globalAppState.isEditing()) {
```

Comment:
on any entity tab click, check if we are currently in edit mode.  if so, confirm with the user that changes may be lost.